### PR TITLE
who got loot from raids

### PR DESCRIPTION
**Pull request by BenDriller**

### CODE REVIEW

File: KTLOSAQ40Loot.json [hunk 1]
```diff
@@ -374,12 +374,12 @@
...
 		},
 		"Qiraji Bindings of Command":{
-			"prio":["Impactt(1)","Dill(1)","Silkyjohnson","Rogues->Non-drake talon DPS Warriors","EP/GP"],
+			"prio":["Silkyjohnson(2)","Rogues->Non-drake talon DPS Warriors","EP/GP"],
 			"gp":"8",
 			"wowID":"20928"
 		},
 		"Qiraji Bindings of Dominance":{
-			"prio":["Kuku","Blacklisted","Austeezy","EP/GP"],
+			"prio":["Kuku","Austeezy","EP/GP"],
```
Comment (zhang165) [inline KTLOSAQ40Loot.json:382]: Austeezy got em already too

File: KTLOSAQ40Loot.json [hunk 2]
```diff
@@ -525,7 +525,7 @@
 			"wowID":"23557"
 		},
 		"Ouro's Intact Hide":{
-			"prio":["Frau(2)","Dill(1)","Aisli(1)","Warrior/rogues","EP/GP"],
+			"prio":["Frau(2)","Aisli(1)","Warrior/rogues","EP/GP"],
```
Comment (zhang165) [inline KTLOSAQ40Loot.json:528]: I think Frau got legs too

File: KTLOSAQ40Loot.json (hunk 1 above)
Comment (BenDriller) [reply KTLOSAQ40Loot.json:382]: okie dokie

File: KTLOSAQ40Loot.json (hunk 2 above)
Comment (BenDriller) [reply KTLOSAQ40Loot.json:528]: I contacted him to confirm

**Pull request merged**